docs/README.md

```diff
@@ -62,6 +62,7 @@ In case the requirements are ambiguous, please write down your assumptions into
 file.
 Please always create a section for each iteration and put your assumptions under this section.
 This helps us to follow certain design decisions in your code which you came up with based on your assumptions.
+Don't hesitate to also explain assumptions using comments in your source code.
```

Comment:
ℹ️ Tiny hint to candidates that they may receive extra kudos for writing code comments... as many candidates skip on writing them.

Reply:
But for assumptions, we have the `assumptions.txt` already.
Would rather have it there than in code or?

Reply:
tend to agree to have them easily discoverable, what about adding that information in the `README.md`

Reply:
🤔 I may have worded this wrongly.

There are things you may need to explain in code that are not assumptions. The comments won't get outdated. There should not be easy discoverability. It does not indicate bad/unreadable code... quite the opposite.

---
Consider these code comments:
- https://github.com/george-labs/george-backend-hiring/compare/main...simon_kocurek_session_9#diff-7f0f4126043d229f4a3afb7093dd558d5477b7ade87b1f683610bbec4e584259R14-R16
- https://github.com/george-labs/george-backend-hiring/compare/main...simon_kocurek_session_9#diff-17d20695fa19eee073b3599e2189d2d8e35557cfecda539ca7c1b6b1d566c1b0R47-R48
- https://github.com/george-labs/george-backend-hiring/compare/main...simon_kocurek_session_9#diff-f06d726559c9c67412a85a3074a29ac4e44e7cca33d968c5ceb26aac24cd7994R15-R16
- https://github.com/george-labs/george-backend-hiring/compare/main...simon_kocurek_session_9#diff-17d20695fa19eee073b3599e2189d2d8e35557cfecda539ca7c1b6b1d566c1b0R31-R32